packages/components/src/components/tooltip/tooltip.stories.ts

```diff
@@ -65,198 +65,62 @@ export const Default = {
   }
 };
 
-export const Placement = {
-  parameters: { controls: { exclude: 'placement' } },
-  render: (args: any) => {
-    return html`<div class="placement-story">
-      ${['top', 'bottom'].map(value =>
-        generateTemplate({
-          axis: {
-            x: {
-              type: 'attribute',
-              name: 'placement',
-              values: [value, `${value}-start`, `${value}-end`]
-            }
-          },
-          args,
-          constants: [
-            {
-              type: 'template',
-              name: 'placement',
-              value: `<div class="template-height template-position">%TEMPLATE%</div>`
-            }
-          ]
-        })
-      )}
-    </div> `;
-  }
+/**
+ * Use the `size` attribute to change the tooltip trigger size.
+ *
+ * - `large` is the default tooltip trigger size.
+ * - `small` can be used as an alternative.
+ */
+export const Size = {
+  render: () => html`
+    <div class="flex items-center gap-12">
+      <sd-tooltip content="Lorem ipsum" placement="bottom" size="lg" trigger="click focus"></sd-tooltip>
+      <sd-tooltip content="Lorem ipsum" placement="bottom" size="sm" trigger="click focus"></sd-tooltip>
+    </div>
+  `
 };
 
-export const Size = {
-  parameters: { controls: { exclude: 'size' } },
-  render: (args: any) =>
-    generateTemplate({
-      axis: {
-        x: {
-          type: 'attribute',
-          name: 'size'
-        }
-      },
-      args,
-      constants: [
-        { type: 'template', name: 'width', value: '<div class="template-position">%TEMPLATE%</div>' },
-        {
-          type: 'attribute',
-          name: 'placement',
-          value: 'bottom'
-        }
-      ]
-    })
+/**
+ * Use the `placement` attribute to set the placement.
+ *
+ * - `top`: displays tooltip above the trigger element
+ * - `top-start`: displays tooltip above the trigger element and aligns arrow to the start of the tooltip container
+ * - `top-end`: displays tooltip above the trigger element and aligns arrow to the end of the tooltip container
+ * - `bottom`: displays tooltip below the trigger element
+ * - `bottom-start`: displays tooltip below the trigger element and aligns arrow to the start of the tooltip container
+ * - `bottom-end`: displays tooltip below the trigger element and aligns arrow to the end of the tooltip container
+ */
+export const Placement = {
+  render: () => html`
+    <sd-tooltip content="Lorem ipsum" placement="bottom" size="lg" trigger="click focus"></sd-tooltip>
+  `
 };
 
+/**
+ * Use the `disabled` attribute to disable the tooltip.
```

Comment:
Your color in the disabled icon is not matching figma. why is that? do we need to open a fix ticket?
In figma we have Neutral-500

Reply:
I've checked and `sd-interactive--disabled` is using `neutral-600`. Do we need to update? If so, we need to open an issue

Reply:
@smfonseca tag me once the disabled tooltip is Neutral-500. I only need this to approve the ticket..

Reply:
This will be done once [1364](https://github.com/solid-design-system/solid/issues/1364) is tackled